CHANGELOG.md

```diff
@@ -1,5 +1,93 @@
 # Changelog
 
+## [v0.19.0](https://github.com/delta-io/delta-kernel-rs/tree/v0.19.0/) (2025-12-19)
+
+[Full Changelog](https://github.com/delta-io/delta-kernel-rs/compare/v0.18.2...v0.19.0)
+
+### 🏗️ Breaking changes
+1. Error on surplus columns in output schema ([#1528])
+2. Remove `arrow-55` support ([#1507])
+3. Add read_parquet_schema API to ParquetHandler ([#1498])
+4. Add `write_parquet_file` to `ParquetHandler` ([#1392])
+5. Make PartialEq for Scalar a physical comparison ([#1554])
+6. Expose mod time in scan metadata callbacks ([#1565])
```

Comment:
Let's add a note here: "You will need to change your scan callback function to just take a struct which has all the previous arguments as members (and the mod time). See an example of the needed change [here](https://github.com/delta-io/delta-kernel-rs/pull/1565/files#diff-3f44d0b7f8cfbe763cbe0cdbb2e2450a84833065de32fc102aef9d38b21b3daaR62). For FFI code, your callback function needs an extra argument. See an example of the change needed [here](https://github.com/delta-io/delta-kernel-rs/pull/1565/files#diff-60493959e34a593831a075fbf2cc7a03a45d8f423f98e2d6b6a4a6ce479dd25bR54).".

Reply:
thanks!!